database.cpp

```diff
@@ -1040,6 +1040,18 @@ static int sqliteLoadConfigCallback(void *user, int ncols, char **colval , char
             }
         }
     }
+    else if (strcmp(colval[0], "group0") == 0)
+    {
+        if (!val.isEmpty())
+        {
+            uint group0 = val.toUInt(&ok);
+            if (ok && (group0 <= 65535))

```

Comment:
Shouldn't 65535 be 0xFFF0 to align with the later code?

Reply:
Yes, good catch,